python/composio/tools/local/codeanalysis/chunker.py

```diff
@@ -1,12 +1,10 @@
-import os
 import re
-import subprocess
 from typing import Any, Dict, List, Tuple, Union
 
 from tree_sitter import Language, Parser
+import tree_sitter_python as tspython
 
-from composio.tools.local.codeanalysis.constants import TREE_SITTER_FOLDER
-
+PY_LANGUAGE = Language(tspython.language())
```

Comment:
Consider adding a docstring or comment explaining the purpose of this module-level constant and why it's initialized at the module level (e.g., for performance reasons).